CHANGELOG.md

```diff
@@ -445,3 +445,28 @@ All notable changes to this project will be documented in this file. The format
 ### Fixed
  - Fix: Certificate, from hostrule CRD, can not be assigned to L7 VirtualService when AKO is deployed in Dedicated mode.
  - Fix: Virtualservices are getting deleted when there is an issue with an access to the kube_api server which results in setting up `deleteConfig` flag to true.
+
+## AKO-1.12.1
+
+### Added
+- AKO now claims support for Kubernetes 1.29, OCP version 4.14.
+- AKO supports LoadBalancerClass for LoadBalancer type services.
+- AKO allows pool member to be IPV6 or IPV4 in dual stack deployment for `CALICO` and `ANTREA` CNI.
+- IPV6 address can be used to connect to the Avi Controller.
+- VRF support within VCenter Cloud
+- AKO now has a `L7Rule` CRD to change default parameters of L7 VirtualService in addition to `Hostrule` CRD.
```

Comment:
Hostrule-> "HostRule"

Reply:
Have we missed adding multiport support for services here ?

Reply:
@pkoshtavmware Can you please give me link for PR `Multiport support for Services` ?

Reply:
https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/1373/files.
My bad, it was only for Gateways not ingresses